fastjet/internal/delaunay/point.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package delaunay
+
+import (
+	"fmt"
+	"math"
+
+	"go-hep.org/x/hep/fastjet/internal/predicates"
+)
+
+// Point in the X-Y Plane.
+//
+// It holds dynamic information about the
+// adjacent triangles, the nearest neighbor and the distance to that neighbor.
+type Point struct {
+	// x and y are the coordinates of the point.
+	x, y float64
+	// adjacentTriangles is a list of triangles containing the point.
+	adjacentTriangles triangles
+	// id is used when points are removed. Copies of the points around the point to
+	// be removed are made. The id is set incremental in counterclockwise order. It identifies
+	// the original the point. The id is then used to determine if a Triangle is inside or outside the
```

Comment:
`the original the point`
there seems to be missing something there (or there's one too many `the`.)